templates/glossary-footer-default.html

```diff
@@ -0,0 +1,318 @@
+{% foreach($scripts as $script): %}
+<script src="{{$script}}" type="text/javascript"></script>
+{% endforeach; %}
+
+<script type="text/javascript">
+    jQuery.fn.exists = function () { return this.length > 0; }
+
+    $(document).ready(function() {
+        var clickedAlphabet = '';
+        var glossary_action = '';
+        var alphabets = Array.from({ length: 26 }, (_, i) => String.fromCharCode(65 + i));
+        $("#divSearchForTerms").hide();
+       
+        alphabets.forEach(function(alphabet) {
+            var button = $('<button type="button" class="btn btn-primary btn-xs">').text(alphabet)
+                            .addClass('alphabet-button')
+                            .attr('data-alphabet', alphabet)
+                            .attr('id', 'btnAlpha')
+                            .attr('name', 'btnAlpha');
+            $('#alphaButtons').append(button);
+        });
+        
+        $('btnAlpha').on('click', function () {
+            // getAlpaTerms()
+
+        });
+
+        $('#glossary-body').on('click', '#action_buttons button, #action_buttons button .fa, #btn_new_term', function (event) {
```

Comment:
Please break this down into 3 click event functions - easier to read and maintain,